### PR TITLE
Hawkular Metrics - 2017 Roadmap

### PR DESCRIPTION
**Pull request by stefannegrea**

### CODE REVIEW

File: src/main/jbake/content/blog/2017/02/07/hawkular-metrics-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+**Roadmap - 2017**
+
+
+* **Administration & Management**
+    * Add monitoring and management features for Cassandra and application server; the goal is to unify the configuration and management for Cassandra, Metrics and Alerting.
+    * Provide diagnostics and error reports to administrators; automatic or on-demand,  scan the setup and find errors to help debug difficult problems
+    * Allow automatic scaling of both Cassandra and application servers via the HPA
+
+
+* **Pre-computted Aggregates (carry-over from 2016)**
```
Comment (jotak) [inline src/main/jbake/content/blog/2017/02/07/hawkular-metrics-roadmap.md]: Pre-computed

**Comment by jotak** [comment]
Could we put links to the related JIRA tickets? Like I found the venerable HWKMETRICS-1 is for aggregations, I didn't find something related to stats query language

File: src/main/jbake/content/blog/2017/02/07/hawkular-metrics-roadmap.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+The project was updated to run on Cassandra 3.x in May ([release 0.15.0](https://github.com/hawkular/hawkular-metrics/releases/tag/0.15.0)). The native Grafana integration was released in July ([release 0.17.0](https://github.com/hawkular/hawkular-metrics/releases/tag/0.17.0)) and it is now an [independent project](https://github.com/hawkular/hawkular-grafana-datasource). And lastly, we improved the developer support by creating distributions that include Metrics, Alerting and Cassandra pre-deployed inside Wildlfly to make it extremely simple to download and get the project running.
+
+The pre-computed aggregates were postponed to this year due to te major work done for compressing data at rest released in October ([release 0.20.0](https://github.com/hawkular/hawkular-metrics/releases/tag/0.20.0)). The histogram metrics were also carried-over to this year.
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2017/02/07/hawkular-metrics-roadmap.md]: due to te major

File: src/main/jbake/content/blog/2017/02/07/hawkular-metrics-roadmap.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+**Roadmap - 2017**
+
+
+* **Administration & Management**
+    * Add monitoring and management features for Cassandra and application server; the goal is to unify the configuration and management for Cassandra, Metrics and Alerting.
+    * Provide diagnostics and error reports to administrators; automatic or on-demand,  scan the setup and find errors to help debug difficult problems
+    * Allow automatic scaling of both Cassandra and application servers via the HPA
```
Comment (pilhuhn) [inline src/main/jbake/content/blog/2017/02/07/hawkular-metrics-roadmap.md]: Explain HPA or mention "HPA on Kubernetes" (or both)

**Comment by stefannegrea** [comment]
@jotak, I do not think we have JIRAs for the rest of the proposed items; but I added the link to the ticket you mentioned.

**Comment by jotak** [comment]
Thanks @stefannegrea , I was curious to see if I could get more info, so if I am I imagine others could be as well :)

**Comment by stefannegrea** [comment]
Is anything still holding this from being merged?

**Comment by pilhuhn** [comment]
@stefannegrea I thought you might want to hold it until 0.24.1 is out and announce this one instead.

**Comment by stefannegrea** [comment]
@pilhuhn, the roadmap needs to be published ahead of the Metrics 0.24.0 release announcement.

**Pull request merged**